ooiservices/app/uframe/controller.py

```diff
@@ -376,64 +375,124 @@ def _compile_glider_tracks():
             try:
                 p_p = r_p.json()
                 for gl in p_p:
-                    glider_location = "/"+p+"/"+gl+"/00-ENG000000/"
+                    glider_location = "/"+p+"/"+gl+"/04-DOSTAM000/"
                     glider_locations.append(base_url+glider_location)
-                    glider_name =  glider_location + 'telemetered/glider_eng_telemetered'
+                    glider_name =  glider_location + 'telemetered/dosta_abcdjm_glider_instrument'
                     url = base_url+glider_name
                     glider_ids.append(url)
             except:
                 print "error:", p, r_p.content
 
+    #check for the update flag, will try and update only those available
+    if update_tracks:
+        glider_cache = cache.get('glider_tracks')
+
+
+    lat_field   = "latitude"
+    lon_field   = "longitude"
+    depth_field = 'int_ctd_pressure'
+    bar_to_m    = 0.09804139432
+
     #params for position and depth info
-    params = "?parameters=PD1335,PD1336,PD1276&limit=1000"
+    #only request "depth" as nominal lat,lon are passed anyway
+    params = "?parameters=PD7&limit=10"
```

Comment:
Interpolation is broken.  We need t revert to PD 1335 and 1336.  Also it doesnt look like you request depth anymore...

Side note, we need the limit to be over 10.  1000 should be good.